app/models/megaphone/episode.rb

```diff
@@ -183,7 +199,7 @@ def cuepoints_batch!(cuepoints)
       # validate all the cuepoints about to be created
       cuepoints.all? { |cp| cp.validate!(:create) }
       body = cuepoints.map { |cp| cp.as_json_for_create }
-      self.api_response = api.put("podcasts/#{podcast.id}/episodes/#{id}/cuepoints_batch", body)
+      self.api_response = api.put_base("episodes/#{id}/cuepoints_batch", body)
```

Comment:
cuepoints for some reason don't include the network in the path, unlike ever other endpoint in the API!